src/modules/project/pages/projectView/views/projectNavigation/ProjectNavigation.tsx

```diff
@@ -38,11 +37,12 @@ export const ProjectNavigation = ({ showLabel }: { showLabel?: boolean }) => {
   const location = useLocation()
   const [_, changeProjectSideNavOpen] = useProjectSideNavAtom()
 
-  const { isProjectOwner, onCreatorModalOpen, project, setMobileView, loading } = useProjectContext()
+  const { isProjectOwner, onCreatorModalOpen, project, setMobileView, loading, inProgressGoals, completedGoals } =
+    useProjectContext()
 
   const { entriesLength, rewardsLength } = useProjectDetails(project)
 
-  const { hasGoals } = useProjectGoals()
+  const hasGoals = inProgressGoals?.length || completedGoals?.length
```

Comment:
You are exporting hasGoals from useProjectContext, maybe import from there instead of the repeated logic here.